### PR TITLE
FIX Jenkins 38492 update result header to show waiting for input state

### PR DESCRIPTION
**Pull request by scherler**
# Description

See [JENKINS-38492](https://issues.jenkins-ci.org/browse/JENKINS-38492).

screenshot from 2016-12-15 14-47-17

See as well https://github.com/jenkinsci/jenkins-design-language/pull/123/files 

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: blueocean-core-js/src/js/components/RunButton.jsx [hunk 1]
```diff
@@ -98,12 +99,16 @@ export class RunButton extends Component {
...
         });
-        const stopLabel = this.state.stopping ? translate('toast.stopping', {
+        let stopLabel = this.state.stopping ? translate('toast.stopping', {
             defaultValue: 'Stopping ...',
         }) : translate('toast.stop', {
             defaultValue: 'Stop',
         });
 
+        if (isPaused && !this.state.stopping) {
+            stopLabel = translate('toast.abort', { defaultValue: 'Abort' });
```
Comment (kzantow) [inline blueocean-core-js/src/js/components/RunButton.jsx]: Not sure why this is `toast.`, as it's nothing to do with toasts.

**Comment by i386** [comment]
@tscherler can you run the PR deployer so that @brody can review ?

**Comment by i386** [comment]
@scherler see my last comment

**Comment by scherler** [comment]
@kzantow you are completely right about the naming but we need to update all keys used in web and core-js to reflect the new naming scheme that @cliffmeyers implemented anyway. I see this in an upcoming PR.

**Comment by scherler** [comment]
@i386 http://pr.sandbox.blueocean.io/pr-660/

File: blueocean-dashboard/src/main/js/components/records.jsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export class RunRecord extends Record({
         if (this.isCompleted()) {
             return this.result;
         } else if (this.state === 'PAUSED') { // TODO: remove this after JDL is defined for paused state
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/records.jsx]: couldn't this if and comment now be removed?

**Comment by michaelneale** [comment]
:bee: LGTM - and passes ATH, unless someone has objections... 

@i386 as an interim for input capturing, should "input requested" be a hyperlink to the classic console ? (this would make input "work" for some people while we wait for the api to be finished).

**Comment by i386** [comment]
if it's a quick thing todo then sure. Otherwise we are not that far away from the real thing

**Comment by michaelneale** [comment]
(I am just anticipating the tickets) - yeah only if quick (in this or another PR, maybe calculating the classic URI will be not worth the effort...)

**Comment by michaelneale** [comment]
actually this seems a :bug: as it isn't showing it in the header for me: 

screen shot 2016-12-16 at 4 24 31 pm

It also shows it as running in the activity and branch listings on the pipeline summary (but that is probably a separate PR)

**Comment by scherler** [comment]
@michaelneale the last screenshot how can I reproduce that?

**Comment by scherler** [comment]
@michaelneale found it, just add a success stage as first node.

**Comment by scherler** [comment]
hmm, 
screenshot from 2016-12-16 10-34-55

**Comment by scherler** [comment]
It is a timing issue. I saw your screenshot once.

**Comment by scherler** [comment]
https://ci.blueocean.io/job/ATH-Jenkinsfile/job/master/660/ passed, will now merge

**Pull request merged**